src/admin-views/events-meta-box.php

```diff
@@ -233,34 +234,25 @@ class='alignleft'
 							class="tribe-dropdown"
 							data-prevent-clear
 						>
-							<?php
-							if ( isset( $_EventCurrencyPosition ) && 'suffix' === $_EventCurrencyPosition ) {
-								$suffix = true;
-							} elseif ( isset( $_EventCurrencyPosition ) && 'prefix' === $_EventCurrencyPosition ) {
-								$suffix = false;
-							} elseif ( true === tribe_get_option( 'reverseCurrencyPosition', false ) ) {
-								$suffix = true;
-							} else {
-								$suffix = false;
-							}
+							<?php // phpcs:ignore -- Legacy naming.
+							$suffix = $_EventCurrencyPosition ? ( 'suffix' === $_EventCurrencyPosition ? true : false ) : ( tribe_get_option( 'reverseCurrencyPosition' ) ?: false );
 							?>
-							<option value="prefix"> <?php _ex( 'Before cost', 'Currency symbol position', 'the-events-calendar' ) ?> </option>
-							<option value="suffix"<?php if ( $suffix ) {
-								echo ' selected="selected"';
-							} ?>><?php _ex( 'After cost', 'Currency symbol position', 'the-events-calendar' ) ?></option>
+							<option value="prefix"> <?php _ex( 'Before cost', 'Currency symbol position', 'the-events-calendar' ); ?> </option>
```

Comment:
```suggestion
							<option value="prefix" selected( ! $suffix )> <?php _ex( 'Before cost', 'Currency symbol position', 'the-events-calendar' ); ?> </option>
```

For accessibility purposes, any selected option should be marked as such. Screen readers will read the selected item even before the select is opened.